src/data/fintype/basic.lean

```diff
@@ -649,6 +649,10 @@ list.length_fin_range n
 @[simp] lemma finset.card_fin (n : ℕ) : finset.card (finset.univ : finset (fin n)) = n :=
 by rw [finset.card_univ, fintype.card_fin]
 
+-- this `fact` is needed as an instance when working with SL₂, for example
```

Comment:
Let's make this a little more explicit:
```suggestion
/-- This `fact` is needed as an instance by `matrix.special_linear_group.has_neg`. -/
```

Reply:
```suggestion
-- This `fact` is needed as an instance by `matrix.special_linear_group.has_neg`.
```
I believe we have a rule that a docstring should not consist _only_ of an implementation note, but -- if it exists at all -- should start with a description of the contents.

Reply:
My main motivation here was for someone looking at doc-gen and thinking "wow, this is a weird lemma to have", which an implementation comment wouldn't be visible to.

Reply:
OK, 
```suggestion
/-- The cardinality of `fin (bit0 k)` is even, `fact` version.
This `fact` is needed as an instance by `matrix.special_linear_group.has_neg`. -/